Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:xenial
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+RUN echo "force-unsafe-io" > /etc/dpkg/dpkg.cfg.d/02apt-speedup
+RUN echo "Acquire::http {No-Cache=True;};" > /etc/apt/apt.conf.d/no-cache
+
+RUN \
+  apt-get update && \
+  apt-get -y install \
+          software-properties-common
+
+RUN apt-get -y install python-pip git postgresql-server-dev-9.5 graphviz
+RUN pip install --upgrade pip
+
+COPY requirements.txt /tmp
+RUN pip install -r /tmp/requirements.txt
+RUN TF_BINARY_URL=https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.11.0rc1-cp27-none-linux_x86_64.whl && pip install --upgrade $TF_BINARY_URL
```

Comment:
I don't usually use docker so I'm not super familiar here, but it's too bad if it precludes GPUs.